webcodecs/audioDecoder-codec-specific.https.any.js

```diff
@@ -152,31 +177,47 @@ promise_setup(async () => {
     '?opus': OPUS_DATA,
     '?pcm_alaw': PCM_ALAW_DATA,
     '?pcm_mulaw': PCM_MULAW_DATA,
+    '?vorbis': VORBIS_DATA,
   }[location.search];
 
+  // Fetch the media data and prepare buffers.
+  const response = await fetch(data.src);
+  const buf = await response.arrayBuffer();
+
+  CONFIG = {...data.config};
+  if (data.config.description) {
+    if (Array.isArray(data.config.description)) {
+      const length = data.config.description.reduce((sum, value) => sum + ((typeof value === 'number') ? 1 : value.size), 0);
```

Comment:
Please describe the format this expects.

Reply:
I've added a brief description and a link to the spec.